content/en/docs/Writing policies/variables.md

```diff
@@ -329,6 +335,16 @@ Reference the image properties of container `tomcat`:
 
 `{{images.containers.tomcat.digest}}`
 
+6. Reference the readable reference for the image (along with the image digest)
+
+`{{images.containers.tomcat.reference}}`
```

Comment:
What's shown above does not include the image digest so I'm confused.

Reply:
`reference` is like "registry/path@digest" but since we rather have tag in our example (they are mutually exclusive), `reference` becomes "registry/path:tag". hence, `reference` is the readable reference for the image (along with the image digest), but when tag is the identifier, rather than digest we use tag.

Reply:
Ok, so the description here shouldn't say "along with the image digest" because that's only in the case where the image was originally specified with a digest and not tag, correct?

Reply:
I mean here I chose a more generic definition. So yes, if we want to be specific, we should change that phrase.